purchase_request_to_rfq/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "author": "Eficent, "
               "Acsone SA/NV,"
               "Odoo Community Association (OCA)",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.1.0",
     "website": "http://www.eficent.com",
```

Comment:
https://github.com/OCA/purchase-workflow

Reply:
Comment attended. Please update review. Thank you!

Reply:
Already Updated review :smile: 

Just rebase and squash commits